boxes/generators/dividertray.py

```diff
@@ -130,6 +137,11 @@ def render(self):
             max(facing_wall_length, side_wall_length), self.h, "ffff", move="right only"
         )
 
+        # Bottom piece.
+        if self.bottom:
+            self.rectangularWall(facing_wall_length, side_wall_length, "ffff",
+                    callback=[partial(self.generate_finger_holes, side_wall_length)], move="up")
```

Comment:
This looks a bit fishy. Shouldn't the edge type be "e" for the sides that don't have a side wall attached? As happens when left_wall or right_wall is False?

Reply:
It looks like you might be right :-).